.vscode/cspell.json

```diff
@@ -1249,6 +1249,13 @@
         "unprovisioning"
       ]
     },
+    {
+      "filename": "**/sdk/recoveryservices/**/*.cs",
+      "words": [
+        "bcdr",
+        "Bcdr"
```

Comment:
I think this is case insensitive, we should only need add one variant.

Reply:
removed